.github/workflows/rust.yml

```diff
@@ -86,6 +104,12 @@ jobs:
     steps:
       - name: "Checkout repo"
         uses: actions/checkout@v4
+      - name: "Checkout maintainer tools"
+        uses: actions/checkout@v4
+        with:
+          repository: tcharding/rust-bitcoin-maintainer-tools
+          ref: 05-02-ci
```

Comment:
Same issue of the collision resistance as the other PR I've commented.

Reply:
Maybe use a `YYYY-MM-DD-ci` approach?

Reply:
These are just my personal branch names (and git tree directories), they should never appear on master in anything. I see others use year prefix but I don't want to type "2024-" a thousand times a day.